### PR TITLE
tests: key_press tweaks

### PR DESCRIPTION
**Pull request by marusak**
Picked from  #12579 as it was getting big.

### CODE REVIEW

File: test/common/testlib.py [hunk 1]
```diff
@@ -245,14 +245,16 @@ def blur(self, selector):
...
         for key in keys:
             args = {"type": "keyDown", "modifiers": modifiers}
 
             # If modifiers are used we need to pass windowsVirtualKeyCode which is
             # basically the asci decimal representation of the key
             args["text"] = key
-            if (not key.isalnum() and ord(key) < 32) or modifiers != 0:
+            if use_ord:
+                args["windowsVirtualKeyCode"] = ord(key)
+            elif (not key.isalnum() and ord(key) < 32) or modifiers != 0:
```
Comment (martinpitt) [inline test/common/testlib.py]: Is there a reason why we still need to support that older implicit mode? This isn't being used a lot.

**Comment by martinpitt** [comment]
Some selenium failures on firefox and chromium, let's at least get these to green.

File: test/common/testlib.py (hunk 1 above)
Comment (marusak) [reply test/common/testlib.py]: There are a lot of `\r`. These characters are represented by this branch.

Comment (martinpitt) [reply test/common/testlib.py]: I see, thanks!

**Pull request merged**